stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/client/InitEndpoint.java

```diff
@@ -137,6 +137,62 @@ private static void populateClientConnectorConfig(MapValue<String, Object> clien
         return customHeaders;
     }
 
+    /**
+     * Populate the retry config.
+     *
+     * @param retryConfig - the retry config.
+     * @param retryConnectorConfig - the retry connector config.
+     */
+    private static void populateRetryConnectorConfig(MapValue<String, Object> retryConfig,
+                                                     RetryContext retryConnectorConfig) {
+        retryConnectorConfig.setInterval(getIntValue(retryConfig, INTERVAL_IN_MILLIS, 1000));
+        retryConnectorConfig.setBackOfFactor(getDoubleValue(retryConfig));
+        retryConnectorConfig.setMaxInterval(getIntValue(retryConfig, MAX_WAIT_INTERVAL, 30000));
+        retryConnectorConfig.setMaxAttempts(getIntValue(retryConfig, MAX_COUNT, 0));
+    }
+
+    /**
+     * Validate and create the webSocket service.
+     *
+     * @param clientEndpointConfig - a client endpoint config.
+     * @param strand - which holds the observer context being started
```

Comment:
The comment https://github.com/ballerina-platform/ballerina-lang/pull/19723#discussion_r372936156 applies to here as well. Check all the places.